tests/types/test_resolvers.py

```diff
@@ -58,3 +77,18 @@ def hello2(self, info, query, limit) -> str:
         'Missing annotation for arguments "limit" and "query" '
         'in field "hello2", did you forget to add it?',
     )
+
+
+# noinspection PyUnusedLocal
+def test_raises_error_when_missing_annotation_and_resolver():
+    with pytest.raises(MissingFieldAnnotationError) as e:
+
+        @strawberry.type
+        class Query:  # noqa: F841
+            missing = strawberry.field(name="annotation")
+
+    [message] = e.value.args
+    assert message == (
```

Comment:
nit: This reads better I think :)

```suggestion
    assert e.value.args[0] == (
```

Reply:
I dunno.... I find the single-item tuple syntax to be pretty bad...

I'll change it though to be consistent

Reply:
I just release in other places we check the whole tuple, which looks nicer 😊

Reply:
Sorry, but I'm not sure what this comment is supposed to mean 😅

Reply:
this: https://github.com/strawberry-graphql/strawberry/blob/3a4b6ec0effd3c86db781059c77618651f9d71dc/tests/types/test_resolvers.py#L34-L36 :)

as opposed to sue my suggestion above with args[0] :)

Reply:
Oh I didn't even notice that the suggestion you proposed _didn't_ do that, cause you did it everywhere else and I guess I just glossed over it. That's why I mentioned the single-item tuple syntax -- that trailing comma is really easy to miss haha.

To clarify, you're suggesting I use the literal tuple (second example), instead of the indexing pattern (first example), right? I'll change it to whichever you want 👍.

Reply:
let's do this then 😊 https://github.com/strawberry-graphql/strawberry/blob/3a4b6ec0effd3c86db781059c77618651f9d71dc/tests/types/test_resolvers.py#L34-L36